### DISCUSSION TITLE
Missing Keybind <leader>cR for File Rename using LazyVim LSP

### DISCUSSION THREAD

**Discussion by Armadillidiid** [LazyVim/LazyVim#1668] [General]
I've recently noticed that the keybind <leader>cR which was previously used for renaming files using the LSP seems to be missing. This keybind was an essential feature for my workflow and its absence is causing inconvenience.

**Comment by folke** [comment]
LSP renames are now done autmatically when using neotree, mini.files for all LSP servers instead of just tsserver
